packages/web-react/src/components/Grid/__tests__/Grid.test.tsx

```diff
@@ -106,4 +98,44 @@ describe('Grid', () => {
     expect(element).not.toHaveStyle({ '--grid-spacing-x-desktop': 'var(--spirit-space-1200)' });
     expect(element).not.toHaveStyle({ '--grid-spacing-y-desktop': 'var(--spirit-space-1200)' });
   });
+
+  it('should render with horizontal alignment', () => {
+    render(<Grid alignmentX="center" data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXCenter');
+  });
+
+  it('should render with default alignment classes', () => {
+    render(<Grid data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXStretch Grid--alignmentYStretch');
+  });
```

Comment:
I'd just put this case before the previous one so it goes like:

1. **default**
2. horizontal
3. horizontal + vertical
4. responsive horizontal
5. responzive horizontal + vertical